src/Microsoft.Fx.Portability/CompressedHttpClient.cs

```diff
@@ -130,6 +128,15 @@ public CompressedHttpClient(ProductInformation info)
                             throw new NotFoundException();
                         case HttpStatusCode.Unauthorized:
                             throw new UnauthorizedEndpointException();
+                        case HttpStatusCode.InternalServerError:
+                            var content = await response.Content?.ReadAsStringAsync();
```

Comment:
This can cause a null ref. If response.Content is null, you'll get 'await null' which will fail.